spring-cloud-gcp-examples/spring-cloud-gcp-trace-example/README.adoc

```diff
@@ -3,23 +3,41 @@
 This sample application demonstrates using the
 link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-trace[Spring Cloud GCP Trace Starter] in your code.
 
+You can read about the Spring Boot Starter for Stackdriver Trace in more detail at the
+https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.BUILD-SNAPSHOT/reference/htmlsingle/#_spring_cloud_sleuth[Spring
```

Comment:
This link will get outdated quickly. Is there some kind "latest" URL?

Reply:
Right, I have the same concern. See my comment.
That is fully unclear why to use external links if we can compile internals...

Reply:
Unfortunately, there is no alternative.
I don't know how we publish the reference document in order to generate a separate "latest" version and it doesn't exist currently.

We can't use internal links because we're linking from a README file to the reference document.

Reply:
How about just refer from this README to the target .adoc relatively instead?